# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmConstants.h
# Commit: 455f2bae38f3
# Full Hash: 455f2bae38f3dec9d30a1aa9025de7b7199f39f7
# Author: Andy Wingo <wingo@igalia.com>
# Date: 2020-03-09 21:52:54
# Regressor Bug: 1620197
# File Overlap Count: 1
# Description:
#   Bug 1620197 - Enable multiple results from WebAssembly functions r=lth
#   
#   This patch enables multi-value calls and returns, adding some tests, and
#   conditionally disabling a couple expect-fail reftests that now pass.
#   
# ==============================================================================

diff -r 6cafdb82131f -r 455f2bae38f3 js/src/wasm/WasmConstants.h
--- a/js/src/wasm/WasmConstants.h	Mon Mar 09 11:54:27 2020 +0000
+++ b/js/src/wasm/WasmConstants.h	Mon Mar 09 11:30:43 2020 +0000
@@ -603,11 +603,6 @@
 static const unsigned MaxMemoryInitialPages = 16384;
 static const unsigned MaxCodeSectionBytes = MaxModuleBytes;
 
-// FIXME: Temporary limit to function result counts.  Replace with MaxResults:
-// bug 1585909.
-
-static const unsigned MaxFuncResults = 1;
-
 // A magic value of the FramePointer to indicate after a return to the entry
 // stub that an exception has been caught and that we should throw.
 